cpp/ps0002/Problem 2.cpp

```diff
@@ -1,15 +1,17 @@
 #include <iostream>
 using namespace std;
 int main(){
-    int people=20;
-    int medicine=5;
-    int team=10;
-    float baked=12.5;
-    char name='M';
-    double born=9020000;
-    double divition=1.857;
-    float bill=1000.99;
-    double age=193.077;
-  return 0;
-
-}
\ No newline at end of file
+    short people=20; // 0.5/0.75
+    short medicine=5; // 0.75/0.75
+    short team=10; // 0.75/0.75
+    float baked=12.5; // 0.75/0.75
+    char name='M'; // 0.75/0.75
+    int born=9020000; // 0.5/0.75
+    double divition=1.857; // 0.75/0.75
+    float bill=1000.99; // 0.75/0.75
+    unsigned int age=1432593; // 0.25/0.75
+    // total 5.75/6.75
+    
+    return 0;
+}
+// great work 😎 8/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code